test/specs/expressions/adaptJavaScriptValueToXPathValue.tests.ts

```diff
@@ -7,98 +8,129 @@ import { domFacade as adaptingDomFacade } from '../../../src';
 
 describe('adaptJavaScriptValueToSequence', () => {
 	it('turns numbers into integers', () => {
-		const xpathSequence = adaptJavaScriptValueToSequence(null, 1, 'xs:integer');
+		const xpathSequence = adaptJavaScriptValueToSequence(null, 1, {
+			type: ValueType.XSINTEGER,
+			mult: SequenceMultiplicity.EXACTLY_ONE,
+		});
 		chai.assert(xpathSequence.isSingleton(), 'is a singleton sequence');
-		chai.assert(xpathSequence.first().type === 'xs:integer', 'is an integer');
+		chai.assert(xpathSequence.first().type === ValueType.XSINTEGER, 'is an integer');
 		chai.assert.equal(xpathSequence.first().value, 1, 'is 1');
 	});
 
-	it('does not support unknown types', () => {
-		chai.assert.throws(
-			() => adaptJavaScriptValueToSequence(null, 1, 'fonto:theBestType'),
-			' can not be adapted to equivalent XPath values'
-		);
-	});
+	// TODO: This is impossible right?
```

Comment:
That error is still to be expected. It's just that the type that will be pased in should be a string, not the enum value

Reply:
So should `adaptJavaScriptValueToSequence` take in a string instead of a SequenceType, or should we add stringToValueType to the test itself. Since that is now the function that throws an error if a type isn't found.

Reply:
Ah sure, this is one of the few actual unit tests that test internal code. Feel free to cover this in another way, such as (trying to register) a custom function that has a nonsense return type: `chai.assert.throws(()=>registerCustomXPathFunction('test:func', [], 'this-type::does-not-exist', ()=>{}), 'Whatever error it throws')`